docs/contributing/pull-request-guidelines/index.md

```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. One is [small change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md) and the other one is [standard change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/standard-change.md). Select one based on the following condition.
```

Comment:
I think you can remove this part and combine it with the below part.

```suggestion
There are two types of template. Select one based on the following condition.
```

Reply:
fixed in [c5dd772](https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/c5dd7721d30e50153005e026bcf93304cd542d6d)